Packs/SIGNL4/Integrations/SIGNL4/SIGNL4.py

```diff
@@ -0,0 +1,95 @@
+from CommonServerPython import *
+
+
+class Client(BaseClient):
+
+    def send_signl4_alert(self, **kwargs):
+
+        payload = {
+            "Title": kwargs.get('json_data').get('title'),
+            "Message": kwargs.get('json_data').get('message'),
+            "X-S4-ExternalID": kwargs.get('json_data').get('s4_external_id'),
+            "X-S4-Status": "new",
+            "X-S4-Service": kwargs.get('json_data').get('s4_service'),
+            "X-S4-Location": kwargs.get('json_data').get('s4_location'),
+            "X-S4-AlertingScenario": kwargs.get('json_data').get('s4_alerting_scenario'),
+            "X-S4-Filtering": kwargs.get('json_data').get('s4_filtering'),
+            "X-S4-SourceSystem": "CortexXSOAR"
+        }
+
+        return self._http_request(method='POST', json_data=payload)
+
+    def close_signl4_alert(self, **kwargs):
+
+        payload = {
+            "X-S4-ExternalID": kwargs.get('json_data').get('s4_external_id'),
+            "X-S4-Status": "resolved",
+            "X-S4-SourceSystem": "CortexXSOAR"
+        }
+
+        return self._http_request(method='POST', json_data=payload)
+
+
+def test_module(client):
+    """
+    Performs basic get request to get item samples
+    """
+    payload = {
+        "title": "Test Alert from Cortex XSOAR",
+        "X-S4-SourceSystem": "CortexXSOAR"
+    }
+    result = client.send_signl4_alert(method='POST', json_data=payload)
+    if 'eventId' in result:
+        demisto.results("ok")
+    else:
+        error_code = result['error_code']
+        description = result['description']
+        demisto.results(f'{error_code} {description}')
+
+
+def send_signl4_alert(client, **kwargs):
+    result = client.send_signl4_alert(**kwargs)
+    return result
```

Comment:
Note that the commands do not return anything to the Context, nor is anything readable displayed to the user.
It is recommended to use `CommandResults`.
https://xsoar.pan.dev/docs/integrations/code-conventions#commandresults

Reply:
Thanks and this will be part of the following commit.